patientsearch/src/js/components/Search.js

```diff
@@ -290,13 +356,15 @@ export default function Search() {
                                         disableFuture
                                         clearable
                                         format="yyyy-MM-dd"
-                                        helperText="(YYYY-MM-DD format)"
+                                        helperText="(YYYY-MM-DD format), example: 1950-01-02"
```

Comment:
```suggestion
                                        helperText="(YYYY-MM-DD format), example: 1977-01-12"
```
Just because I always forget Luke Skywalker's DOB :)

Reply:
Fixed, :-)

Reply:
:lightsaber: :)